packages/InteractiveProfilingTool-Tests.package/IPTModelTestCase.class/instance/buildButton.st

```diff
@@ -0,0 +1,4 @@
+helpers
+buildButton
```

Comment:
I don't think this method simplifies the code in `testBuildButton` enough to justify the complexity increase of having another method. Also you need to read this method when trying to undersand what `testBuildButton` does exactly. Same goes for `buildWindow`.
If you were to remove these two methods you could also get rid of the category `helpers`, which would also simplify the `IPTModelTestCase` class.